src/core/components/layout/components/columns/styles.ts

```diff
@@ -1,18 +1,18 @@
 import { css } from '@emotion/react';
 import { space } from '@guardian/src-foundations';
-import { until } from '@guardian/src-foundations/mq';
+import { Breakpoint, from, until } from '@guardian/src-foundations/mq';
 
 export const columns = css`
 	box-sizing: border-box;
 	display: flex;
-	& > * + * {
+	& > div:nth-of-type(n + 2) {
 		margin-left: ${space[5]}px;
 	}
 `;
 
 const collapseBelowSpacing = css`
 	display: block;
-	& > * + * {
+	& > div:nth-of-type(n + 2) {
```

Comment:
I don't quite follow this change. Would you mind explaining what the difference is?

Reply:
Oh yes, I was messing around with something, but I didn't need it in the end. I'll restore the old way. Thanks for asking!